<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TickerNeighbors</class>
 <widget class="QDialog" name="TickerNeighbors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>186</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Industries</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineTicker"/>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="pushTicker">
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/query.png</normaloff>:/images/query.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboSector"/>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="pushSector">
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/query.png</normaloff>:/images/query.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboIndustry"/>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="pushIndustry">
       <property name="icon">
        <iconset resource="tht.qrc">
         <normaloff>:/images/query.png</normaloff>:/images/query.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkNyse">
       <property name="text">
        <string>NYSE</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkNasd">
       <property name="text">
        <string>NASD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="checkAmex">
       <property name="text">
        <string>AMEX</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QListView" name="list">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::TopToBottom</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushCopy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Found:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="labelFound"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineTicker</tabstop>
  <tabstop>pushTicker</tabstop>
  <tabstop>comboSector</tabstop>
  <tabstop>pushSector</tabstop>
  <tabstop>comboIndustry</tabstop>
  <tabstop>pushIndustry</tabstop>
  <tabstop>checkNyse</tabstop>
  <tabstop>checkNasd</tabstop>
  <tabstop>checkAmex</tabstop>
  <tabstop>pushCopy</tabstop>
 </tabstops>
 <resources>
  <include location="tht.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushCopy</sender>
   <signal>clicked()</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotCopy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineTicker</sender>
   <signal>returnPressed()</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>22</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkNyse</sender>
   <signal>toggled(bool)</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkNasd</sender>
   <signal>toggled(bool)</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkAmex</sender>
   <signal>toggled(bool)</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushTicker</sender>
   <signal>clicked()</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSector</sender>
   <signal>clicked()</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushIndustry</sender>
   <signal>clicked()</signal>
   <receiver>TickerNeighbors</receiver>
   <slot>slotFetch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCopy()</slot>
  <slot>slotFetch()</slot>
 </slots>
</ui>
